instructure/canvas-ios
# Share extension UI update

refs: MBL-15728
affects: Student
release note: Updated UI on the file share component. To submit your files to assignments directly from iOS, tap the share icon and select the Student app.

test plan:
- Log out with all users or reset cache in the application.
- Start the file share extension.
- Please log in via the application message should be shown.
---
- Log in with a student in the app then start the share extension with an image and a video.
- Course list should populate upon entering the course selector screen.
- Assignments of the selected course should show up in the assignments list screen.
- Add a text comment.
- Check the preview of the shared images, their list should be horizontally scrollable, video length should be indicated.
- Tap on submit, UI should disappear and submission success message should be shown in a notification.
---
- Create an assignment which has a file linked in its description.
- Open the student app, go to that assignment, open the file. It should be shown in a separate screen.
- Start the share extension.
- The previously visited course and assignment should be auto selected.
---

![IMG_40A438BB8171-1](https://user-images.githubusercontent.com/72396990/141076979-6d95add9-a08e-4f08-aec2-9f9496f020af.jpeg)
